data-prepper-api/src/main/java/org/opensearch/dataprepper/expression/ExpressionEvaluator.java

```diff
@@ -21,8 +21,6 @@
      * @param statement string to be parsed and evaluated
      * @param context event used to resolve external references in the statement
      * @return coerced result of statement evaluation
-     *
-     * @throws ExpressionEvaluationException if unable to evaluate or coerce the statement result to type T
```

Comment:
It seems that we should move this exception into Data Prepper API as well. And then add this back.